src/mudlet-mapper.xml

```diff
@@ -3500,7 +3500,7 @@ mmp.echo("We're connected to StickMUD.")</script>
 						<colorTriggerFgColor>#000000</colorTriggerFgColor>
 						<colorTriggerBgColor>#000000</colorTriggerBgColor>
 						<regexCodeList>
-							<string>^The door to the (\w+) is closed forcefully fromthe other side and the sound of a bolt being slammed home makes your lack of admittance clear\.$</string>
+							<string>^The door to the (\w+) is closed forcefully from the other side and the sound of a bolt being slammed home makes your lack of admittance clear\.$</string>
```

Comment:
Are you sure about this - it is a game text - could it be miss-formatted like this if it is broken at the Game Server end?

Reply:
Good point, but I'd guess the typo is fixed by now.